### DISCUSSION TITLE
Version release for 8.x branch

### DISCUSSION THREAD

**Discussion by akshaykatyal** [ruflin/Elastica#2212] [General]
By when will the tag for the 8.x branch release?

**Comment by ruflin** [comment]
The simple answer would be, when all the tasks on that list are resolved: https://github.com/ruflin/Elastica/issues/2185 But not sure how quickly we get to it. I assume we already have quite a few people on the 8.x branch and have not heard so far much complaints.

@akshaykatyal Did you have a chance to test 8.x on your end? Any issue / unexpected behaviour?

**Reply by akshaykatyal** [reply]
Yes, I have tested the 8.x branch and didn't find any issues for the search query topics useful for me if the changelog is followed for migration. But still initial tag release for 8.x branch like 8.0.alpha will be appreciated.

**Reply by ruflin** [reply]
Good callout. I just pushed 8.0.0-alpha1 tag. Let me know if this doesn't help: https://github.com/ruflin/Elastica/tree/8.0.0-alpha1

**Reply by thePanz** [reply]
@akshaykatyal good news, v8.0.0 just got released!
\cc @ruflin

**Comment by ruflin** [comment]
Yep, release is out: https://github.com/ruflin/Elastica/releases/tag/8.0.0 Going to close this discussion.
